quadrat/child-theme.json

```diff
@@ -306,7 +306,7 @@
 				},
 				"citation": {
 					"typography": {
-						"fontSize": "15px",
+						"fontSize": "16px",
```

Comment:
This should use the variable var(--wp--preset--font-size--small)

Reply:
Fixed in https://github.com/Automattic/themes/pull/4683/commits/ced4e066263f375c82add6f20754151878155c4b